asyncpg_listen/listener.py

```diff
@@ -169,3 +178,32 @@ def _push(_: Any, __: Any, channel: Any, payload: Any) -> None:
             queue.put_nowait(Notification(channel, payload))
 
         return _push
+
+    async def _process_notification(self, handler: NotificationHandler, notification: NotificationOrTimeout) -> None:
+        if self._tracer is None:
+            self._tracer = opentelemetry.trace.get_tracer(__package__)
+        if self._meter is None:
+            self._meter = opentelemetry.metrics.get_meter(__package__)
+        if self._notification_histogram is None:
+            self._notification_histogram = self._meter.create_histogram("asyncpg_listener_notification")
+
+        if isinstance(notification, Timeout):
+            name = f"Notification timeout #{notification.channel}"
+        elif isinstance(notification, Notification):
+            name = f"Notification #{notification.channel}"
```

Comment:
what if move this iff-else block to separate function `get_notification_name(notification) -> str` ?

Reply:
Fixed